ieeeuoft/hackathon-template
# Send Decision Emails to Participants

## Overview

- Resolves #184 
- Creates a template for sending emails using the django admin site
- Creates a link in the top bar of the admin site when in `Review --> Teams` that takes you to the page to send decisions. Decisions can only be sent by superusers
- Can see number of Accepted, Waitlisted, Rejected people left to send emails to

## Unit Tests Created

- Check permissions for only superusers allowed to access page
- Send emails to Accepted, Waitlisted, Rejected and check that the correct amount are sent
- Send emails to Accepted, Waitlisted, Rejected and check that the text is correct in the subject and body

## Steps to QA

- Create some users and reviews and send various emails to them using the system. Make sure you do this as a superuser. 
- You should be able to see the emails in your console. Make sure the text is correct based on the status of the users you are sending emails for
- Make sure the number of emails sent corresponds to the number of emails you entered to send on the form
- Try sending with a date constraint and make sure it only sends emails to those users that were reviewed within those days.
